extension/src/web-component-service.js

```diff
@@ -0,0 +1,52 @@
+const fetch = require('node-fetch')
```

Comment:
You can put this class into `utils` or `service` folder.

Reply:
IMO putting some into folders is not helpful if you have a small-sized project like extension module, I did not think current structure is also right, I am considering we will change those structures a little bit later.

Reply:
OK let's do this change later when we have more files, so we know how to structure better.